data/successors/successor ships.txt

```diff
@@ -150,7 +152,7 @@ ship "Ejniisde"
 		`"Chyyra" Inductive Reverser`
 		"Hyperdrive"
 
-	gun 0 -106
+	pylon 0 -106
```

Comment:
I made that switch on the chance that they took the same view as some human ships do: Better to have a missile that can scare someone away or chase the enemy while the civilian runs away than to have a gun that is only useful if one is turning to face and engage the enemy. 

The description doesn't really specify whether or not this ship is controlled by a faction that has access to missile-type weapons, only that it doesn't generally have armament itself.

If civilians don't have access to any sort of weaponry that would utilize pylons, and  those who do have access to such weapons would generally not use this ship, then yes, it would make sense that it would not have a pylon.